views/js/controller/creator/views/subsection.js

```diff
@@ -81,6 +83,11 @@ define([
         if (!_.isEmpty(config.routes.blueprintsById)) {
             subsectionModel.hasBlueprint = true;
         }
+
+        if (servicesFeatures.isVisible('taoQtiTest/creator/properties/selectionWithReplacement', false)) {
+            sectionModel.hasSelectionWithReplacement = true;
```

Comment:
Do we need to initialise `hasSelectionWithReplacement` to avoid undefined cases?

Reply:
thats correct, but according to the template engine (handlebars) thats just fine. This behaviour was confirmed on other cases of the same view 
https://handlebars-archive.knappi.org/builtin_helpers.html

"You can use the if helper to conditionally render a block. If its argument returns false, undefined, null, "", 0, or [], Handlebars will not render the block."

Reply:
updated to fill the value and reduce the amount of if sentences :), thanks @gabrielfs7 !